src/main/java/org/opentripplanner/middleware/controllers/api/BugsnagController.java

```diff
@@ -0,0 +1,43 @@
+package org.opentripplanner.middleware.controllers.api;
+
+import org.opentripplanner.middleware.bugsnag.BugsnagDispatcher;
+import org.opentripplanner.middleware.bugsnag.EventSummary;
+import org.opentripplanner.middleware.bugsnag.response.Project;
+import org.opentripplanner.middleware.models.BugsnagEvent;
+import org.opentripplanner.middleware.persistence.Persistence;
+import org.opentripplanner.middleware.persistence.TypedPersistence;
+import org.opentripplanner.middleware.utils.JsonUtils;
+import spark.Request;
+import spark.Response;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * Responsible for providing the current set of Bugsnag events to the calling service
+ */
+public class BugsnagController {
+
+    private static TypedPersistence<BugsnagEvent> bugsnagEvents = Persistence.bugsnagEvents;
+
+    /**
+     * Get all Bugsnag events from Mongo and replace the project id with the project name and return
+     */
+    public static String getEventSummary(Request request, Response response) {
+        response.type("application/json");
+
+        List<EventSummary> eventSummaries = new ArrayList<>();
+        List<BugsnagEvent> events = bugsnagEvents.getAll();
+        HashMap<String, Project> projects = BugsnagDispatcher.getProjects();
+
+        // FIXME: Group by error/project type?
+        for (BugsnagEvent event : events) {
+            Project project = projects.get(event.projectId);
```

Comment:
This could result in an NPE if there is a projectId for a project that was created after system startup. I'm not sure how often new projects might be created, but it might be worth at least accounting for this possibility.

Reply:
@evansiroky - The NPE is handled in EventSummary, but I can cover here if you think that is better? 

I've decided to add another job to pull the projects from Bugsnag every minute. I can see this becoming an issue in the future if the otp-middleware is rarely restarted and new projects are added to Bugsnag. Running every minute is probably overkill, but this is in sync with the event requests and configurable.